shared/components/_button.scss

```diff
@@ -94,3 +94,31 @@
 %nio-button--danger {
   @include nio-button--colorize($palette-danger, $palette-white);
 }
+
+%nio-button--huge {
+  font-size: 1.5rem;
+  padding: .6rem;
+  min-height: 2.75rem;
+  min-width: 10rem;
+}
+
+%nio-button--large {
+  font-size: 1.2rem;
+  padding: .4rem;
+  min-height: 2.25rem;
+  min-width: 8.5rem;
+}
+
+%nio-button--small {
+  font-size: .9rem;
+  padding: .2rem;
+  min-height: 1.75rem;
+  min-width: 5.5rem;
+}
+
+%nio-button--tiny {
+  font-size: .7rem;
+  padding: .2rem;
```

Comment:
This padding on tiny seems a little large to me. Maybe we can make all the padding based off a ratio from a standard?